File: Code/Cplus/LAB06_03/LAB06_03.cpp
#include <iostream>
using namespace std;

int main() {
    // สร้าง Array สำหรับเก็บจำนวนทศนิยม 20 ตัว
    double numbers[20];
    double subtractionResult = 0;
    double multiplicationResult = 1;

    // รับจำนวนจากผู้ใช้
    cout << "Enter 20 decimal numbers:" << endl;
    for (int i = 0; i < 20; i++) {
        cout << "Number " << i + 1 << ": ";
        cin >> numbers[i];
    }

    // คำนวณรวมของผลลบ (ตัวที่ 1 - ตัวที่ 2 - ... - ตัวที่ 20)
    subtractionResult = numbers[0]; // เริ่มจากตัวแรก
    for (int i = 1; i < 20; i++) {
        subtractionResult -= numbers[i];
    }

    // คำนวณผลรวมของคูณ (ตัวที่ 1 * ตัวที่ 2 * ... * ตัวที่ 20)
    for (int i = 0; i < 20; i++) {
        multiplicationResult *= numbers[i];
    }

    // แสดงผลจำนวนที่ผู้ใช้ป้อน
    cout << "\nNumbers entered:" << endl;
    for (int i = 0; i < 20; i++) {
        cout << "Number " << i + 1 << ": " << numbers[i] << endl;
    }

    // แสดงผลการลบและผลการคูณ
    cout << "\nResult of subtraction: " << subtractionResult << endl;
    cout << "Result of multiplication: " << multiplicationResult << endl;

    return 0;
}

File: Code/Python/LAB06_03.py
def main():
    # สร้าง list สำหรับเก็บจำนวนทศนิยม 20 ตัว
    numbers = []

    # รับจำนวนจากผู้ใช้
    print("Enter 20 decimal numbers:")
    for i in range(1, 21):
        number = float(input(f"Number {i}: "))
        numbers.append(number)

    # คำนวณรวมของผลลบ (ตัวที่ 1 - ตัวที่ 2 - ... - ตัวที่ 20)
    subtraction_result = numbers[0]  # เริ่มจากตัวแรก
    for i in range(1, len(numbers)):
        subtraction_result -= numbers[i]

    # คำนวณผลรวมของคูณ (ตัวที่ 1 * ตัวที่ 2 * ... * ตัวที่ 20)
    multiplication_result = 1
    for number in numbers:
        multiplication_result *= number

    # แสดงผลจำนวนที่ผู้ใช้ป้อน
    print("\nNumbers entered:")
    for i, number in enumerate(numbers, start=1):
        print(f"Number {i}: {number}")

    # แสดงผลลัพธ์การลบและผลลัพธ์การคูณ
    print(f"\nResult of subtraction: {subtraction_result}")
    print(f"Result of multiplication: {multiplication_result}")

if __name__ == "__main__":
    main()
